### DISCUSSION TITLE
Pull Shark Achievement

### DISCUSSION THREAD

**Discussion by akbarjorayev** [community/community#122328] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I think I closed 130 PRs. Here it is:
<img src="https://github-readme-stats.vercel.app/api?username=akbarjorayev&show_icons=true&title_color=294fd9&icon_color=294fd9" />

But I still do not have Silver (x3) level. Why?

**Comment by davevad93** [comment, marked as answer]
Hi @akbarjorayev ,

Maybe not all your `pull requests` were merged, I mean, to obtain the silver `Pull Shark` achievement you need 128 `pull requests` merged and the total PR's count lists also the ones that have been closed without being merged. For example my total count of PRs is 25 but I've merged only 21 in fact I've recieved the bronze `Pull Shark` achievement with 21 total PRs so that's probably why you haven't received the achievement yet.

**Reply by 0xabdulkhaliq** [reply]
Yeah you're right @davevad93 !

@akbarjorayev, It only shows "**Total PRs**" not "**Merged PRs**", the Pull Shark x3 needs 128 Merged Pull Requests.

For example, take a look at my status:

![Abdul Khalid's GitHub stats](https://github-readme-stats.vercel.app/api?username=0xabdulkhalid&show_icons=true)

I have more than 500 PRs in Total, but out of them only 425 has been accepted and Merged!

So i got this [Pull Shark x3](https://github.com/0xabdulkhalid?achievement=pull-shark&tab=achievements) badge as a result.

In order to get claim x3 badge you need to open PRs which are acceptable by community so that they can be merged.

**Reply by akbarjorayev** [reply]
How did you know exactly merged PRs? There is no number `425`

**Reply by 0xabdulkhaliq** [reply]
@akbarjorayev, It's simple!

You can head to your [Pull Requests](https://github.com/pulls) page, you can change the `is:open` into `is:merged` to know how many of your PRs are merged. (or) you can try this [URL](https://github.com/pulls?q=is%3Amerged+is%3Apr+author%3Aakbarjorayev+archived%3Afalse+) to count your Merged ones.

For example, Take a look at mine

![image](https://github.com/community/community/assets/92252895/5cef1eaf-b2b5-4d6c-8836-784f0daf75a7)

PS: Assalamu Alaikum Warahmatullahi Wabarakatuh Brother!

**Comment by akbarjorayev** [comment]
> @akbarjorayev, It's simple!
> 
> You can head to your [Pull Requests](https://github.com/pulls) page, you can change the `is:open` into `is:merged` to know how many of your PRs are merged. (or) you can try this [URL](https://github.com/pulls?q=is%3Amerged+is%3Apr+author%3Aakbarjorayev+archived%3Afalse+) to count your Merged ones.
> 
> For example, Take a look at mine
> 
> ![image](https://private-user-images.githubusercontent.com/92252895/328252473-5cef1eaf-b2b5-4d6c-8836-784f0daf75a7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUwMTQ0NTAsIm5iZiI6MTcxNTAxNDE1MCwicGF0aCI6Ii85MjI1Mjg5NS8zMjgyNTI0NzMtNWNlZjFlYWYtYjJiNS00ZDZjLTg4MzYtNzg0ZjBkYWY3NWE3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA2VDE2NDkxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwYzI0YzI0MThiMDAwMDJjMjMyOTY0Zjg5ZTA0OTU4MTRlMGMxMGI2MWE2ZWQ1MWE1ODg1YTZhZTI1YzViNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kN-sXJs0v7VAzLcZsBvKYYkyxEogGAWHwObZiUUQkWQ)
> 
> PS: Assalamu Alaikum Warahmatullahi Wabarakatuh Brother!


Thanks ❤️
